roles/maintenance_10_linux/defaults/main.yml

```
@@ -51,3 +51,5 @@ linux_allowed_large_files:
   - "/var/log/lastlog"
 
 linux_additional_allowed_large_files: []
+
+excluded_logs_per_host: []
```

Please prefix the defaults with the role name:

```suggestion
linux_excluded_logs_per_host: []
```